rq/job.py

```diff
@@ -108,7 +121,7 @@ def cancel_job(job_id: str, connection: 'Redis', serializer=None, enqueue_depend
     Job.fetch(job_id, connection=connection, serializer=serializer).cancel(enqueue_dependents=enqueue_dependents)
 
 
-def get_current_job(connection: Optional['Redis'] = None, job_class: Optional['Job'] = None) -> Optional['Job']:
+def get_current_job(connection: Optional["Redis"] = None, job_class: Optional["Job"] = None) -> Optional["Job"]:
```

Comment:
One more thing, do you kind reverting the changes from single to double quotes? If you use black, please run it with —skip-string-normalization

Reply:
Oh didn't realize that. Sure we can recert the quotes

Reply:
Hmmm I may have messed up this. Let me try to fix it tomorrow to do things properly

Reply:
It seems like the original version mixes single and double quotes. What is a good way to properly revert only lines that changed the quotes? I've tried to add a temporary ruff linting config to fix the quotes, but some quotes in the file are double quotes.

Reply:
I did it for you by reverting everything and conditionally committing only the quotes revert, I also added a suggestion of using `typing_extensions` that I mentioned in the issue you initially created https://github.com/rq/rq/issues/2091

If you merge https://github.com/Microwave-WYB/rq/pull/1 it will apply those suggestions to this PR